webapp/src/components/call_widget/component.tsx

```diff
@@ -187,7 +187,7 @@ export default class CallWidget extends React.PureComponent<Props, State> {
                 height: '28px',
                 borderRadius: '4px',
                 color: '#D24B4E',
-                background: 'rgba(210, 75, 78, 0.04)',
+                background: 'rgba(210, 75, 78, 0.08)',
```

Comment:
This (leave button background) is unrelated but it wasn't matching the design although I could find differing versions. One with 8% (barely visible) and one with 24%. @abhijit-singh Could you point me to the most up to date value?

![image](https://user-images.githubusercontent.com/1832946/174610295-405cd2a4-42dc-45ee-8979-47bb113bd70b.png)

Reply:
Ideally, it should be 8% in light themes and 24% in dark themes. But afaik that's not simple to implement so I'd recommend going with 8% right now, and then we'll work on a solution for opacities like this under a broader initiative around updating colour variables across different themes.